capstone/src/main/webapp/youtubeScript.js

```diff
@@ -174,3 +198,75 @@ function isLetter(character) {
     return false;
   }
 }
+
+async function getTrending(categoryId) {
+  const trendingResponse = await fetch('/trending_servlet?videoCategoryId=' + categoryId,)
+  const trendingResponseJson = await trendingResponse.json();
+  const trendingVideoIds = [];
+  for (const item in trendingResponseJson.items) {
+    const videoId = trendingResponseJson.items[item].id;
+    trendingVideoIds.push(videoId);
+  }
+  const commentsList = []
+  for (const id in trendingVideoIds) {
+    const videoCommentList = await fetch('/youtube_servlet?videoId=' + trendingVideoIds[id],)
+    const videoCommentListJson = await videoCommentList.json();
+    commentsList.push(videoCommentListJson);
+  }
+  inputCommentsToPerspective(commentsList);
+}
+
+function enableTextInput(button) {
+  if (button.checked) { 
+    document.getElementById('channelIdForAnalysis').value = button.id;
+    document.getElementById('channelIdForAnalysis').disabled = true;   
+  }
+}
+
+function disableTextInput(button) {
+  if (button.checked) { 
+    document.getElementById('channelIdForAnalysis').value = "";
+    document.getElementById('channelIdForAnalysis').disabled = false;
+  }
+}
+
+/** Creates radio buttons to allow teh user to select between various categories*/
+function showCategories() {
+  /** Creates button to enable manual input*/
+  const radiobox = document.createElement('input');
+  radiobox.type = 'radio';
+  radiobox.id = 'manualInput';
+  radiobox.value = 'manualInput';
+  radiobox.name = 'same';
+  radiobox.checked  = true;
+  const label = document.createElement('label');
+  label.htmlFor = 'manualInput';
+  const description = document.createTextNode('ID/Username');
+  label.appendChild(description);
+  radiobox.onclick = function() {
+    disableTextInput(this);   
+  }
+  const container = document.getElementById('category-container');
```

Comment:
nit: Let's rename `container` to something a little more descriptive like `categoryContainer`

Reply:
addressed in "address Chris's nits"